src/app/shared/audio/extendable-audio.ts

```diff
@@ -13,12 +13,12 @@ export class ExtendableAudio {
   ondurationchange: Function;
   ontimeupdate: Function;
   onseeking: Function;
-  paused: boolean;
   volume: number;
 
   constructor(url) {
     if (typeof window['Audio'] !== 'undefined') {
       this._audio = new Audio();
+      this._audio.preload = 'none';
```

Comment:
looks like skipping the preloading gives you a `dovetail-audio.ts:265 Uncaught (in promise) DOMException: Failed to load because no supported source was found.` in the console every time you go to the next track in a playlist

Reply:
Can you elaborate on that?  Are you paused or playing when you click "Next >"?  Or are you just clicking on the episode?

Reply:
I'm seeing that error in the console (but the audio goes on to play) anytime I click 'Next' whether or not I'm paused; also happens when I click on an episode.